plugins/onex/hooks/scripts/pre_tool_use_task_boundary_tests.sh

```
@@ -0,0 +1,125 @@
+
+mkdir -p "$(dirname "$LOG_FILE")"
+
+export PYTHONPATH="${PROJECT_ROOT}:${PLUGIN_ROOT}/lib:${HOOKS_LIB}:${PYTHONPATH:-}"
+
+if [[ -f "$PROJECT_ROOT/.env" ]]; then
+    set -a
+    source "$PROJECT_ROOT/.env" 2>/dev/null || true
+    set +a
+fi
+
+source "${HOOKS_DIR}/scripts/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./scripts/common.sh was not specified as input (see shellcheck -x).

(SC1091)